### PR TITLE
Applied 6 decimals for all money inputs in Back Office

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Once and for all, as prices are stored with 6 decimals in Back Office, lets display it the right way |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-912 |
| How to test? | Every money type in back office, for Product page now have 6 decimals. |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -1616,10 +1633,12 @@ var priceCalculation = (function() {
       var finalPrice = obj.closest('div[id^="combination_form_"]').find('.final-price');
       var defaultFinalPrice = finalPrice.attr('data-price');
       var priceToBeChanged = new Number(price) + new Number(defaultFinalPrice);
+      priceToBeChanged = truncateDecimals(priceToBeChanged, 6);
 
-      finalPrice.html(priceToBeChanged.toFixed(2));
+      finalPrice.html(priceToBeChanged);
     },
     'impactTaxExclude': function(obj) {
+      console.log('yo');
```
Comment (Shudrum) [inline admin-dev/themes/default/js/bundle/product/form.js]: To remove please :)

File: admin-dev/themes/default/template/controllers/orders/form.tpl [hunk 2]
```diff
@@ -351,7 +351,7 @@
 				id_product: id_product,
 				id_product_attribute: id_product_attribute,
 				id_customer: id_customer,
-				price: new Number(new_price.replace(",",".")).toFixed(4).toString()
+				price: ps_round(new Number(new_price.replace(",",".")), 6).toString(),
```
Comment (Shudrum) [inline admin-dev/themes/default/template/controllers/orders/form.tpl]: Are you sure that the last coma on an object work in every browser without transpilation?

Comment (mickaelandrieu) [inline admin-dev/themes/default/template/controllers/orders/form.tpl]: I don't know,  the feature has been tested both on chrome and firefox browser, in ubuntu and Windows 7 operating systems.

**Comment by rGaillard** [comment]
@mickaelandrieu the product page is currently broken with this PR.

**Comment by maximebiloe** [comment]
It my fault, I forgot a "." with the conflict resolution

**Pull request merged**